src/dosbox.cpp

```diff
@@ -781,19 +781,22 @@ void DOSBOX_Init()
 	secprop->AddInitFunction(&DMA_Init);
 	secprop->AddInitFunction(&VGA_Init);
 	secprop->AddInitFunction(&KEYBOARD_Init);
+	secprop->AddInitFunction(&PCI_Init); // PCI bus
 
-	secprop=control->AddSection_prop("pci", &PCI_Init, false); // PCI bus
-
-	secprop->AddInitFunction(&VOODOO_Init, false);
+	secprop = control->AddSection_prop("voodoo", &VOODOO_Init, false);
 
 	const char* voodootypes[] = { "12mb", "4mb", "disabled", nullptr };
-	pstring = secprop->Add_string("voodoo", only_at_start, "12mb");
+	pstring = secprop->Add_string("voodoo_memsize", only_at_start, "12mb");
 	pstring->Set_values(voodootypes);
-	pstring->Set_help("RAM amount of emulated Vodooo 3dfx card.");
+	pstring->Set_help("Memory size of the emulated Vodooo 3dfx card ('12mb' by default).");
```

Comment:
I think we should refrain from the word "emulated" in the config setting descriptions and in the documentation in general as it's very redundant (this an emulator... 😅). I think we should only use "emulated" when we must disambiguate a concept that can refer to both something in the emulated and the host machine (e.g. emulated CPU vs host CPU), but that's not the case here.

Reply:
3dfx Voodoo

Reply:
fully agree! (and dropped)